metadata-models/src/main/pegasus/com/linkedin/dataset/FieldLineageDetails.pdl

```diff
@@ -0,0 +1,18 @@
+namespace com.linkedin.dataset
+
+import com.linkedin.common.Urn
+
+/**
+ * Details of the parent fields and transformation resulting in new field.
+ */
+record FieldLineageDetails {
+    transform: optional string
```

Comment:
Please add documentation for the fields `transform` and `parentFields` :-)

Reply:
Added in latest commit. (Mostly just a rehash of the record level comment.)